bedrock-migration/values.yaml

```
@@ -0,0 +1,9 @@
+imEnabled: "true" #should this be a global value? Is there a better way to determine this?
+
+#stand-in global values for testing
+global:
+  operatorNamespace: "cs-op"
+  instanceNamespace: "cs-serv"
+  tetheredNamespaces: [""]
```

Hi Ben, CPD changed the tethered namespace to `tetheredNamespace` without `s`: https://github.ibm.com/Mengdie-Chu/non-olm-installer/blob/main/top-override.yaml#L7

```suggestion
  tetheredNamespace: [""]
```